public/v1/apps/mongodb.json

```diff
@@ -28,7 +28,7 @@
             "label": "MongoDB Version",
             "defaultValue": "4",
             "description": "Checkout their docker page for the valid tags https://hub.docker.com/r/library/mongo/tags/",
-            "validRegex": "/^([a-zA-Z0-9])+$/"
+            "validRegex": "/^(\d+\.)?(\d+\.)?(\*|\d+)$"
```

Comment:
But the problem is the regexp, it does not allow me to input dots and dash. 
My application is a machine that prevents the alien invasion that needs mongodb 3.4.10, but caprover does not allow me create that specific version because of regex issue. Here is the screenshoot,

![image](https://user-images.githubusercontent.com/177265/51448223-f4f05300-1d5f-11e9-94bc-54b3459faa27.png)


If this is not fix immediately i guess your end of the world for typo is inevitable. :)